File: SSUI/src/app/models/story.ts
import { Ssuser } from './ssuser';

export interface Story {
  ssId: number;
  ssTitle: string;
  ssDescription: string;
  createdOn: Date;
  isApproved: boolean;
  like: number;
  dislike: number;
  id?: string
  ssUser?: Ssuser;
}

File: SSBOL/Story.cs
﻿using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace SSBOL
{
   public class Story
    {
        [Key]
        public int SSId { get; set; }

        [Required]
        public string SSTitle { get; set; }

        [Required]
        public string SSDescription { get; set; }
        public DateTime CreatedOn { get; set; }
        public bool IsApproved { get; set; }
        public int Like { get; set; }
        public int Dislike { get; set; }

        [ForeignKey("SSUser")]
        public string Id { get; set; }

        public SSUser SSUser { get; set; }
    }

 }